packages/exo/test/heap-classes.test.js

```diff
@@ -280,37 +283,41 @@ test('raw guards', t => {
       return 'hello';
     },
     rawIn(obj) {
-      t.is(Object.isFrozen(obj), false);
+      t.false(Object.isFrozen(obj));
       return obj;
     },
     rawOut(obj) {
-      t.is(Object.isFrozen(obj), true);
+      t.true(hardenIsFake || Object.isFrozen(obj));
```

Comment:
Curious why this is the only `Object.isFrozen` case where we have to condition on `hardenIsFake`

Reply:
I count 9 total. 5 affirmative. 4 negative. Perhaps you mean something else?

Reply:
I see some `Object.freeze` calls above in the same test that don't get the same treatment, and arguably I didn't look closely at what the test does, I just saw those lines modified, but not get an extra `hardenIsFake` condition, and was just wondering why.

Reply:
Your intuition was onto something, anyway. I went through each of these cases again and found 4 that did not require the relaxation for `harden.isFake`, left over from when I relaxed these tests to accommodate the unsafe mode of `@endo/harden` where `isFrozen` is not rigged. Will have to revisit if we go so far.